From 388c179f016a11d401dd6bab91e41cbf12003ede Mon Sep 17 00:00:00 2001
From: Eric Paris <eparis@redhat.com>
Date: Thu, 22 Jan 2015 11:36:21 -0500
Subject: [PATCH] Use docker.service not docker.socket in unit files

Some distros, include RHEL and Fedora, are doing away with the docker
socket by default in systemd units, for security reasons.  Instead rely
on the docker.service being started instead of socket activation.
---
 contrib/init/systemd/kubelet.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/init/systemd/kubelet.service b/contrib/init/systemd/kubelet.service
index 52b5b85..6990f29 100644
--- a/contrib/init/systemd/kubelet.service
+++ b/contrib/init/systemd/kubelet.service
@@ -1,8 +1,8 @@
 [Unit]
 Description=Kubernetes Kubelet Server
 Documentation=https://github.com/GoogleCloudPlatform/kubernetes
-After=docker.socket cadvisor.service
-Requires=docker.socket
+After=docker.service cadvisor.service
+Requires=docker.service
 
 [Service]
 EnvironmentFile=-/etc/kubernetes/config
-- 
2.1.0

